generators/app/templates/src/config/i18n.js

```diff
@@ -1,12 +1,18 @@
-import { init } from 'i18next';
+import i18next from 'i18next';
+import { initReactI18next } from 'react-i18next';
 
 function requireAll(requireContext) {
   return requireContext.keys().map(requireContext);
 }
 
-init({
+i18next.use(initReactI18next).init({
+  fallbackLng: 'en',
```

Comment:
Why is the fallback 'en', can we leave it in 'es'?

Reply:
i can change it but makes no sense because the default language is `es`

Reply:
So why do we need a default and fallback? I'm assuming that most projects will only have spanish, so I'd change it to `es`